vita3k/gui/src/common_dialog.cpp

```diff
@@ -50,7 +50,8 @@ static void draw_ime_dialog(DialogState &common_dialog, float FONT_SCALE) {
             common_dialog.ime.max_length + 1);
     }
     ImGui::SameLine();
-    if (ImGui::Button(common_dialog.lang.common["submit"].c_str())) {
+    auto &common = common_dialog.lang.common;
+    if (ImGui::Button(common["submit"].c_str()) || ImGui::IsKeyPressed(static_cast<ImGuiKey>(emuenv.cfg.keyboard_button_cross))) {
```

Comment:
That's really not a "more detailed error message", could you put it in a different commit?

Reply:
Okay, Mr. Macdu.

Reply:
Done